docusaurus/docs/adding-custom-environment-variables.md

```diff
@@ -6,54 +6,49 @@ sidebar_label: Environment Variables
 
 > Note: this feature is available with `react-scripts@0.2.3` and higher.
 
-Your project can consume variables declared in your environment as if they were declared locally in your JS files. By
-default you will have `NODE_ENV` defined for you, and any other environment variables starting with
-`REACT_APP_`.
+Your project can consume variables declared in your environment as if they were declared locally in your JS files. By default you will have `NODE_ENV` defined for you, and any other environment variables starting with `REACT_APP_`.
+
+> WARNING: Do not store any secrets (such as private API keys) in your React app!
+>
+> Environment variables are embedded into the build, meaning anyone can view them by inspecting your app's files. Instead, you should send requests to your backend (which stores your secrets) and send back the responses to your React app.
```

Comment:
```suggestion
> Environment variables are embedded into the build, meaning anyone can view them by inspecting your app's files.
```

I think this can be removed so as to not be specific about recommendations for dealing with secrets.